scene/3d/physics_body.cpp

```diff
@@ -92,17 +136,42 @@ void PhysicsBody::remove_collision_exception_with(Node* p_node) {
 	PhysicsServer::get_singleton()->body_remove_collision_exception(get_rid(),physics_body->get_rid());
 }
 
+void PhysicsBody::_set_layers(uint32_t p_mask) {
```

Comment:
Doesn't ``ObjectTypeDB::bind_method(_MD("_set_layers","mask"),&PhysicsBody::_set_layers);`` expose it to GDScript? And well, having a print each time a deprecated function is used is the point of deprecation warnings :D It's to encourage users to use the new functions instead so that we can eventually remove the old function.